File: A15function/A13PracticeQuestion/A16BinaryToDecimal.java
package A15function.A13PracticeQuestion;

public class A16BinaryToDecimal {

    public static int BinToDec(int BinNum) {
        int pow = 0;
        int decNum = 0;

        while (BinNum > 0) {
            int lastNum = BinNum % 10;
            decNum = decNum + lastNum * ((int) Math.pow(2, pow));
            pow++;
            BinNum = BinNum / 10;
        }
        return decNum;
    }

    public static void main(String args[]) {
        System.out.println(BinToDec(11011));
    }

}

File: A15function/A13PracticeQuestion/A16BinaryToDecimal.js
function binToDec(binNum = 11100) {
  let pow = 0;
  let dec = 0;
  while (binNum > 0) {
    let lastNum = binNum % 10;
    dec = dec + lastNum * Math.pow(2, pow);
    binNum = Math.round(binNum / 10);
    pow++;
  }
  console.log(dec);
}
binToDec();
